packages/esm-appointments-app/src/constants.ts

```diff
@@ -3,3 +3,51 @@ export const basePath = '/appointments';
 export const spaBasePath = `${window.spaBase}/home`;
```

Comment:
Future refactoring:

1. This should probably be called `spaHomePage`
2. The definition should be `${window.getOpenmrsSpaBase()}home` (`getOpenmrsSpaBase()` is normalised so it always returns a path ending in `/`. `window.spaBase` is a direct representation of whatever was fed into the build the process, and thus may or may not contain a trailing slash).